TravelAgent/TravelAgent/Startup.cs

```diff
@@ -37,6 +43,73 @@ public void ConfigureServices(IServiceCollection services)
             {
                 configuration.RootPath = "ClientApp/build";
             });
+
+            //Autentifikacija SingUp
+            services.AddAuthentication(CookieAuthenticationDefaults.AuthenticationScheme);
+
+            services.AddIdentity<Employee, Role>()
+                .AddEntityFrameworkStores<AppDbContext>()
+                .AddDefaultTokenProviders();
+
+            services.Configure<IdentityOptions>(options =>
+            {
+                // Password settings
+                options.Password.RequireDigit = true;
+                options.Password.RequiredLength = 8;
+                options.Password.RequireNonAlphanumeric = false;
+                options.Password.RequireUppercase = true;
+                options.Password.RequireLowercase = false;
+                options.Password.RequiredUniqueChars = 6;
+
+                // Lockout settings
+                options.Lockout.DefaultLockoutTimeSpan = TimeSpan.FromMinutes(30);
```

Comment:
lockout'ą nustatyčiau ant vienos minutės, just in case užblokinsim dirbdami per daug acc :D